spec.bs

```diff
@@ -1842,27 +1853,31 @@ and a [=real time reporting contributions map=] |realTimeContributionsMap|:
           [=get direct from seller signals for a seller=] given |topLevelDirectFromSellerSignals|.
         1. Set |topLevelDirectFromSellerSignalsRetrieved| to true.
       1. If |compWinnerInfo|'s [=leading bid info/leading bid=] is not null, then run
-        [=score and rank a bid=] with |auctionConfig|, |reportingContextMap|[|component|],
+        [=score and rank a bid=] with |auctionConfig|, |reportingContextMap|[auctionConfig],

```

Comment:
after changing these to top level config, it seems the spec no longer has reporting info set for component auction's reporting context?